pkg/sqlite/load.go

```diff
@@ -77,6 +77,9 @@ func NewSQLLiteLoader(outFilename string) (*SQLLoader, error) {
 	if _, err = db.Exec(createTable); err != nil {
 		return nil, err
 	}
+	if _, err := db.Exec("PRAGMA foreign_keys = ON", nil); err != nil {
```

Comment:
So the foreign keys in the definition before weren't active? I thought we've been relying on foreign keys to help validate relationships on insert?